aat/src/resources/features/F-018/S-004.td.json

```diff
@@ -0,0 +1,27 @@
+{
+  "title": "Should return a negative response for a malformed caseID",
+  "_guid_": "S-004",
+  "_extends_": "F-018_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is to add an activity for the case created above",
+    "contains a malformed caseID"
+  ],
+
+  "request": {
+    "pathVariables": {
+      "cid": "@£  $%^&*  (@£$%  ^&*("
+    }
+  },
+    "expectedResponse": {
+      "responseCode": 400,
+      "headers": {
+        "_extends_": "Common_400_Response_Headers"
+      },
+      "body": {
+        "message": "[[ANYTHING_PRESENT]]"
```

Comment:
Please check the message, if practical.

Reply:
As it is returning 201 response, I can not check the message. But for the time being I deleted body in S-004.td.json and inherited Common_400_Response object so it returns "Bad Request" message.